app/views/compute_resources/show.html.erb

```diff
@@ -51,7 +51,7 @@
     </div>
   <% end %>
   <div id="compute_profiles" class="tab-pane">
-    <table class="<%= table_css_classes 'table-two-pane table-fixed' %>">
+    <table class="<%= table_css_classes 'table-two-pane table-fixed' %>" id="two_pane_content">
```

Comment:
...then it won't work. But it didn't work before either :smile: (not that I would test it before but the code already searches for the two-pane classes globally)